rpcs3/Gui/MainFrame.cpp

```diff
@@ -73,7 +73,7 @@ MainFrame::MainFrame()
 	, m_sys_menu_opened(false)
 {
 
-	SetLabel(wxString::Format(_PRGNAME_ " " _PRGVER_ " " RPCS3_GIT_VERSION));
+	SetLabel(wxString::Format(_PRGNAME_ " " RPCS3_GIT_VERSION));

```

Comment:
@Nekotekina I don't think there's really anything wrong with the script. It appends the tag, commits since last tag and first 8 characters of the commit hash. But we for some reason also append the program version.

It would fix the given problem:
![doubleversion](https://cloud.githubusercontent.com/assets/1827708/10267955/679334fa-6ab1-11e5-8db6-6e557390eec9.png)